packages/blade/src/components/Form/FormLabelText.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { VisuallyHidden } from '~components/VisuallyHidden';
+import { Text } from '~components/Typography';
+import BaseText from '~components/Typography/BaseText';
+import { getPlatformType, useBreakpoint } from '~utils';
+import Box from '~components/Box';
+import { useTheme } from '~components/BladeProvider';
+
+type FormLabelTextProps = {
+  position?: 'top' | 'left';
+  neccessityIndicator?: 'required' | 'optional' | 'none';
+  accessibillityText?: string;
+  children: React.ReactNode;
+  id: string;
+};
+
+const FormLabelText = ({
+  position = 'top',
+  neccessityIndicator = 'none',
+  accessibillityText,
+  children,
+  id,
+}: FormLabelTextProps): React.ReactElement => {
+  const { theme } = useTheme();
+  const { matchedDeviceType } = useBreakpoint({ breakpoints: theme.breakpoints });
+  const isDesktop = matchedDeviceType === 'desktop';
+  const isReactNative = getPlatformType() === 'react-native';
+
+  // TODO: replace with <Text /> when #548 merges
+  const neccessityLabel =
```

Comment:
Suggestion: breakdown the conditional, nested ternary makes it harder to follow

Reply:
Will do.